tests/modules/nf-core/ashlar/input_sheet.csv

```diff
@@ -0,0 +1,3 @@
+sample,file_list,args
```

Comment:
Do we still need to provide this file for the module?

Reply:
You're correct, we don't need that now.  Thanks!
I've removed it.